ultralist/screen_printer.go

```diff
@@ -101,6 +102,21 @@ func (f *ScreenPrinter) formatDue(due string, isPriority bool, completed bool) s
 
 }
 
+func (f *ScreenPrinter) formatInformation(todo *Todo) string {
+	var information []string
+	if todo.IsPriority {
+		information = append(information, "*")
+	} else {
+		information = append(information, " ")
+	}
+	if todo.HasNotes() {
+		information = append(information, "N")
+	} else {
+		information = append(information, " ")
+	}
+	return white.Sprint(strings.Join(information, " "))
+}
+
```

Comment:
should we also have a status for archived?

Reply:
Yeah, sure, that's an good idea.

Reply:
Added the A flag and re-merged with master after #180.